usmqe_tests/api/tendrl/others/test_user.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf8 -*-
+"""
+REST API test suite - user
+"""
+
+from usmqe.api.tendrlapi import user as tendrlapi_user
+import json
+
+
+"""@pylatest default
+Setup
+=====
+
+Prepare USM cluster accordingly to documentation.
+
+Further mentioned ``APIURL`` points to: ``http://USMSERVER/api/1.0``.
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_user_get(valid_session_credentials):
+    """@pylatest api/user.get
+    API-users: get user
+    *******************
+
+    .. test_metadata:: author mkudlej@redhat.com dahorak@redhat.com fbalak@redhat.com
+
+    Description
+    ===========
+
+    Get user admin.
+    """
+    test = tendrlapi_user.ApiUser(auth=valid_session_credentials)
+    """@pylatest api/user.get
+    .. test_step:: 2
+
+        Send **GET** request to ``APIURL/users``.
+
+    .. test_result:: 2
+
+        List of users in database is returned
+    """
+    test.users()
+    """@pylatest api/user.get
+    .. test_step:: 3
+
+        Get user info.
+
+        Send **GET** request to ``APIURL/users/admin``.
+
+    .. test_result:: 3
+
+        User information for user *admin* is returned.
+    """
+    test.user('admin')
+
+
+def test_user_get_not_found(invalid_session_credentials):
+    """@pylatest api/user.get_nonexistent
+    API-users: get nonexistent user
+    *******************************
+
+    .. test_metadata:: author dahorak@redhat.com fbalak@redhat.com
+
+    Description
+    ===========
+
+    Get users.
+    """
+    test = tendrlapi_user.ApiUser(auth=invalid_session_credentials)
```

Comment:
I think this incorrect test. Here we should test if there is correct answer for getting info about user who is not in DB. We don't test if we cannot authenticate to API.